experimental/python/mobilebert_model.py

```diff
@@ -0,0 +1,79 @@
+import wget
+import argparse
+import os
+import tensorflow as tf
+import tarfile
+import numpy as np
+
+MODEL_DIR = "/tmp/chrome/mobilebert"
+MODEL_URL = "https://storage.googleapis.com/cloud-tpu-checkpoints/mobilebert/mobilebert_squad_savedmodels.tar.gz"
+OUTPUT_DIR = "../android/app/src/main/assets"
+
+def prepare_model(opts):
+  #download the model
+  if not os.path.exists(MODEL_DIR):
+    os.makedirs(MODEL_DIR)
+  saved_model_tar = os.path.join(MODEL_DIR, 'mobilebert_squad_saved.tar.gz')
+  
+  if not os.path.exists(saved_model_tar):
+    wget.download(MODEL_URL, saved_model_tar)
+
+  #extract
+  tar_file = tarfile.open(saved_model_tar)
+  tar_file.extractall(path=MODEL_DIR)
+
+  #convert to tflite
+  model_path = os.path.join(MODEL_DIR, "mobilebert_squad_savedmodels/quant_saved_model")
+  print(model_path)
+  converter = tf.lite.TFLiteConverter.from_saved_model(model_path)
+  tflite_model = converter.convert()
+  with open(os.path.join(opts.out_dir, "text_mobilebert.tflite"), "wb") as f:
+    f.write(tflite_model)
+
+# def evaluate(opts):
```

Comment:
Do we need this commented out code?

Reply:
experimental subdirectory is a playground for now and I think it doesn't need review. I mentioned it [here](https://github.com/mehrdadh/chrome-on-device-ml/blob/app/experimental/README.md).